config/locales/tr.yml

```diff
@@ -6,5 +6,8 @@ tr:
     settings:
       add_issue_matchers: İş eşleştiricileri ekleyiniz
       issues_matching_will_require_time_entry: Eşleşen iş kayıtlarında  zaman kaydı girilmesi zorunlu olacak
+      exclude_roles: Rolleri zorunlu alanı doldurma zorunluluğundan hariç tutu
+      check_assignee_only: Only match for assignee
```

Comment:
We can say here "Sadece atanan ile eşleştir"

Reply:
thanks again. updated on next commit